opencode-plugin/src/index.ts

```
@@ -0,0 +1,260 @@
+        async execute({ command }) {
+          const disabledFile = join(superegoDir, ".disabled");
+
+          switch (command) {
+            case "status":
+              if (!existsSync(superegoDir)) {
+                return "Superego not initialized. Use 'superego init' to set up.";
+              }
+              if (existsSync(disabledFile)) {
+                return "Superego initialized but DISABLED. Use 'superego enable' to re-enable.";
+              }
+              const hasPrompt = existsSync(join(superegoDir, "prompt.md"));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)